app/static/js/event_list.js

```diff
@@ -5,6 +5,34 @@ $(document).ready(function(){
     $("#rsvpBtn").click(function(){
         rsvpForEvent($("#rsvpBtn").val())
     })
+    var viewPastEventsToggle = $("#viewPastEventsToggle");
+  
+
+    viewPastEventsToggle.prop("checked", false);
+    toggleRows(false);  
+
+    viewPastEventsToggle.prop("checked", g_isPastTerm);
+    if (g_isPastTerm){
+      toggleRows(true)
+      viewPastEventsToggle.prop("disabled", true);
+    }
+      
+    viewPastEventsToggle.on("change", function(){
+      var isChecked = $(this).prop("checked");
+      toggleRows(isChecked);
+
+      localStorage.setItem("toggleState", isChecked ? "checked" : "unchecked")
+    });
+
+    function toggleRows(isChecked) {
+      var tableRows = $(".showlist");
+      if (isChecked){
+        tableRows.show();
+      }
```

Comment:
let's keep the `else` on the same line as the end brace, according to our coding style